### PR TITLE
Add databases curriculum with RDBMS

### PR DESCRIPTION
**Pull request by dirkjonker**

### CODE REVIEW

**Comment by eamelink** [comment]
Good start. The whole 'RDBMS' repetition is a bit awkward; but maybe it's unavoidable. I guess the idea is that people can substitute their own RDBMS of choice? So that if one can do it with Postgres or MariaDB or with Oracle, it's okay?

I think the 'RDBMS SQL basics' is a bit too big; I think if you'd take out joins into a separate topic, it'll be halved in size. 

The "RDBMS SQL advanced techniques" shows very much to the left, that's a bit awkward since based on it's complexity it should be more to the right. Maybe it should depend on 'RDBMS Indexes' and 'RDBMS Constraints'?

Of a more general note; I see myself going to 'abilities' all the time to figure out what the topic is about. I think most of the content should be moved from 'abilities' to the description of the topic, and the 'abilities' section should contain much more detailed abilities.

For example, the ability _Set operations_ is not really an ability. If _Set operations_ is moved to the description, an ability could be (for example) _Use INTERSECT to find values that exist in two different query resultsets_

**Comment by dirkjonker** [comment]
I tried splitting it up a bit and updating the abilities. It is however quite a large subject, it would be better to discuss this with a larger group in front of a whiteboard.

File: core-curriculum/knowledge/databases.json [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+    },
+    {
+      "id": "rdbms-sql-set-operations",
+      "name": "SQL set operations",
+      "description": "Understand the concepts of set theory and how they are used in relational databases",
+      "tags": [],
+      "dependencies": [
+        "rdbms-sql-basics"
+      ],
+      "resources": [
```
Comment (nMoncho) [inline core-curriculum/knowledge/databases.json]: Resource contribution: http://www.sqljoins.com/sql-joins-explained-step-by-step/

Comment (dirkjonker) [reply core-curriculum/knowledge/databases.json]: Good article, thanks! Please feel free to commit any additional resources to the branch yourself

**Pull request merged**